camel-k-runtime-knative/pom.xml

```diff
@@ -91,4 +91,21 @@
         </dependency>
     </dependencies>
 
+    <build>
+        <plugins>
```

Comment:
Is it possible we put this plugin into parent pom.xml?

Reply:
not every artefact requires indexing